index.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
 
-var _           = require('lodash');
-var compile     = require("node-elm-compiler").compileToString;
-var loaderUtils = require("loader-utils");
-var fs          = require("fs");
+var _                   = require('lodash');
+var compile             = require("node-elm-compiler").compileToString;
+var findAllDependencies = require('node-elm-compiler').findAllDependencies;
```

Comment:
If we're going to require the same thing twice so we can pull out each function, we might as well just require node-elm-compiler as `compiler` and use the functions off that object instead.